test/tests/functional/pbs_acl_host_moms.py

```diff
@@ -52,40 +52,40 @@ def setUp(self):
 
         TestFunctional.setUp(self)
 
-        if len(self.moms) != 2:
-            self.skip_test('test requires two MoMs as input, ' +
-                           '  use -p moms=<mom1>:<mom2>')
+        if len(self.moms) != 1:
+            self.skip_test('test requires one MoM as input, ' +
+                           '  use -p moms=<mom>')
```

Comment:
Ok, but in its current form, running this test will leave an active hook behind, which might disturb the behavior of other tests, and any other jobs in general which are submitted to the system after this test is run. Isn't that dangerous?
There's at least one existing test which cleans up hooks after creating them: https://github.com/PBSPro/pbspro/blob/master/test/tests/functional/cgroups_tests.py#L166